src/microbe_stage/systems/OsmoregulationAndHealingSystem.cs

```diff
@@ -141,7 +141,7 @@ protected override void Update(float delta, in Entity entity)
         /// </summary>
         private void ApplyATPDamage(CompoundBag compounds, ref Health health, ref CellProperties cellProperties)
         {
-            if (compounds.GetCompoundAmount(atp) > 0)
+            if (compounds.GetCompoundAmount(atp) > 0.1f)
```

Comment:
Why is this needed? I think this may have massive implications on taking ATP damage when trying to move.

Reply:
This was changed as ATP never reached a low enough level for it to do damage. The bar would look empty but no damage would be done. Should've probably set it to something lower though.

Reply:
Adjusting when the ATP is taken would be a better choice. That's what's been done in the past to ensure cells can be damaged (for example making this run before whatever system applies ATP damage, but after the process system).

Reply:
I ended up changing this to `0.05f`, as I couldn't get this to work otherwise. I think it's sensible to set it to that, as the difference at that point is barely noticeable and I don't think it would have any major implications.

Reply:
Could you add a comment before this if to explain the number? Or actually I think it'd be pretty nice to make it a constant in Constants and add a summary comment explaining the reason for it there.